### PR TITLE
Webpack + Refactor into es6 modules

### PR DESCRIPTION
**Pull request by lynques**
https://github.com/forbes/ad-inspector/issues/7
- Webpack to bundle/minify js and css
- Refactor to es6 modules
- Retain existing functionality

### CODE REVIEW

**Review by annezhou920 — APPROVED** [review]

File: src/loader.js [hunk 1]
```diff
@@ -8,413 +8,262 @@
...
-	}
+/**
+ * Returns a style object to be used for highlighting an element
+ * @param {HTMLElement} element
+ */
+const getHighlightStyle = (element) => {
+    const style = getOffset(element);
+    const padding = 10;
+    for (let k in style) {
+        style[k] = Math.max(0, style[k] - padding) + 'px';
```
Comment (mcscrambles) [inline src/loader.js]: template literal

File: src/loader.js [hunk 2]
```diff
@@ -8,413 +8,262 @@
...
+    }
+    style.width = Math.min(document.width || document.body.offsetWidth, element.offsetWidth + padding * 2) + 'px';
```
Comment (mcscrambles) [inline src/loader.js]: template literal

File: src/loader.js [hunk 3]
```diff
@@ -8,413 +8,262 @@
...
+    style.height = Math.min(document.height || document.body.offsetHeight, element.offsetHeight + padding * 2) + 'px';
```
Comment (mcscrambles) [inline src/loader.js]: template literal

File: src/loader.js [hunk 4]
```diff
@@ -8,413 +8,262 @@
...
-			var targetkeys = googletag.pubads().getTargetingKeys();
-			var targetArr = [];
+/**
+ * Builds an unordered list based off of the collected properties of an ad slot
+ * @param {Object} obj
+ */
+const ulBuilder = (obj) => {
+    const ulDom = utils.dom('ul');
+    for (let key in obj) {
+        const keyDom = utils.dom('b', {}, {}, key + ': ');
```
Comment (mcscrambles) [inline src/loader.js]: template literal

File: src/slot-helper.js [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+    }
+    return params;
+};
+
+/**
+ * Returns link to creative and line item
+ * @return {String}
+ */
+const getLink = (account, type, id) => {
+    return 'https://www.google.com/dfp/' + account + '#delivery/' + type + 'Detail/' + type + 'Id=' + id;
```
Comment (mcscrambles) [inline src/slot-helper.js]: template literal

File: src/slot-helper.js [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+ * @return {Array}
+ */
+const getSlotSizes = (slot) => {
+    const size_array = [];
+    const sizes = slot.getSizes();
+    for (let i = 0; i < sizes.length; i++) {
+        if (typeof sizes[i] == 'string') {
+            size_array.push(sizes[i]);
+        } else {
+            size_array.push(sizes[i].getWidth() + 'x' + sizes[i].getHeight());
```
Comment (mcscrambles) [inline src/slot-helper.js]: template literal

**Review by mcscrambles — CHANGES_REQUESTED** [review]

**Review by mcscrambles — APPROVED** [review]

**Pull request merged**